ophyd/controls/signal.py

```diff
@@ -285,6 +285,13 @@ def _connected(self, pvname=None, conn=None, pv=None, **kwargs):
             self._ses_logger.debug(msg)
 
     @property
+    def connected(self):
+        if self._write_pv is None:
+            return self._read_pv.connected
+        else:
+            return self._read_pv.connected and self._write_pv.connected
```

Comment:
It is mostly a style thing and matters more if you have patterns like

```python
if thing:
    return
else:
    lots
    of
    code
```

If there is a return in and `if` block, then anything under it is the `else` block